core/quorum_initiator.go

```diff
@@ -440,9 +449,15 @@ func (c *Core) finishConsensus(id string, qt int, p *ipfsport.Peer, status bool,
 	}
 }
 
-func (c *Core) connectQuorum(cr *ConensusRequest, addr string, qt int) {
+func (c *Core) connectQuorum(cr *ConensusRequest, addr string, qt int, pledgetokens float64) {
 	c.startConsensus(cr.ReqID, qt)
-	p, err := c.getPeer(addr)
+	var p *ipfsport.Peer
+	var err error
+	if cr.Type == 1 {
```

Comment:
Seems you are doing the same operation in both if and else blocks.

Reply:
Removed the if else blocks